test/controllers/HomeControllerSpec.scala

```diff
@@ -18,7 +18,7 @@ class HomeControllerSpec extends FrontEndTestHelper {
       status(home) mustBe OK
```

Comment:
There are some unnecessary parenthesis that Intellij is complaining about at lines 62 to 64, if you could tidy up aswell?